test/integration/attachment.test.ts

```diff
@@ -0,0 +1,134 @@
+import { describe, before, test } from 'mocha'
+import { Express } from 'express'
+import { expect } from 'chai'
+
+import createHttpServer from '../../src/server'
+import { get, post, postFile } from '../helper/routeHelper'
+
+import Database from '../../src/lib/db'
+
+const db = new Database().db()
+
+describe('attachment', () => {
+  const size = 100
+  const blobData = 'a'.repeat(size)
+  const filename = 'test.pdf'
+  const jsonData = { key: 'test', filename: 'JSON attachment test' }
+  let app: Express
+
+  before(async () => {
+    app = await createHttpServer()
+  })
+
+  afterEach(async () => {
+    await db.attachment().del()
+  })
+
+  describe('invalid requests', () => {
+    test('returns 500 when attempting to retrieve not UUID', async () => {
```

Comment:
if the route controller changes `@Path() id: string,` -> `@Path() id: UUID` then OpenAPI will catch that it's not a UUID and return a 400. And doesn't let you put a non-UUID on Swagger
![image](https://user-images.githubusercontent.com/40722025/226868275-8d2b525e-f9c2-43f2-9c1d-cf7be45a963b.png)